### ISSUE TITLE
rebar2 support

### ISSUE THREAD

**Issue by NikolaBorisov** [Nordix/eredis#39]
Hello,

I'm trying to use Nordix/eredis in rebar2 based project. When I tried to just switch from wooga/eredis to Nordix/eredis in my rebar.config file I started getting the following compile error:

```
./rebar compile
==> eredis (compile)
ERROR: Failed to load app file ebin/eredis.app: {error,
                                                 {error,
                                                  {8,file_io_server,
                                                   invalid_unicode}}}
```

Does Nordix/eredis support being used by erlang projects that still use rebar2?

**Comment by zuiderkwast** [comment]
We haven't thought about the possibility that someone still uses rebar 2. :-) But I we should make it possible to build with any tool, e.g. erlang.mk, mix, rebar 2/3, if possible.

The problem seems to be our non-ASCII names `{maintainers,["Björn Svensson", "Viktor Söderqvist"]}`. They're UTF-8 in src/eredis.app.src but rebar makes them latin1 in ebin/eredis.app. Then the app file fails to load since Erlang assumes that it's UTF-8.

I think this is a bug of rebar 2, but I guess we can just skip the dots on "ö" and just write our names with "o". @bjosv?

**Comment by bjosv** [comment]
> I think this is a bug of rebar 2, but I guess we can just skip the dots on "ö" and just write our names with "o". @bjosv?

Works just fine with me

**Comment by zuiderkwast** [comment]
Fixed by #40
